styles.css

```diff
@@ -0,0 +1,144 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: Courier, sans-serif;
+}
+
+header {
+    background-image: url(mobile_code_image.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+    width: 100%;
+    min-height: 100vh;
+}
+
+.container {
+    margin: 0 auto;
+    width: 80%;
+}
+
+.nav-main {
+    position: fixed;
+    overflow: hidden;
+}
+
+li {
+    list-style: none;
+    margin-right: 10px;
+    text-align: right;
+}
+
+.nav-buttons a:link, .nav-buttons a:visited {
+    color: #FFFFFF;
+    display: inline-block;
+    padding: 15px;
+    text-decoration: none;
+}
+
+.nav-buttons a:hover {
+    color: #000000;
+    display: inline-block;
+    padding: 15px;
+    text-decoration: none;
+}
+
+section {
+    min-height: 100vh;
+    padding: 0 20px;
+}
+
+.image-icons {
+    max-width: 150px;
+}
+
+.footer-nav-bar {
+    float: left;
+    margin: 0 auto;
+    width: 25%;
+    border: 5px #FFFFFF;
+    display: inline-block;
+    position: relative;
+    text-align: center;
+    padding: 0 60px;
+}
+
+.footer-links {
+    overflow: hidden;
+    position: relative;
+    display: inline-block;
+    width: 80%;
+    min-height: 30vh;
+}
+
+
+
+
+/*Tablet */
+
+@media only screen and (min-width: 600px) {
+    header {
+        background-image: url(tablet_code_image.jpg);
+    }
+    li {
+        list-style-type: none;
+        margin-right: 10px;
+        text-align: left;
+    }
+    .footer-links {
+        overflow: hidden;
```

Comment:
remove this to fix the linkedin logo being cut off at bottom of screen when screen resized